apim-ha.yaml

```diff
@@ -977,14 +970,14 @@ Mappings:
        Ubuntu1804: ami-09b49c54b9dd01f63
   WSO2APIMAMIRegionMap:
     ap-southeast-2:
-      Ubuntu1804: ami-07a3bd4944eb120a0
```

Comment:
please update WSO2PuppetMasterRegionMap with the given AMI IDs.

Reply:
fixed